recipe/meta.yaml

```diff
@@ -58,13 +61,10 @@ outputs:
 
   - name: ariadne-codegen-with-websockets
     build:
-      noarch: python
+      noarch: generic
     requirements:
-      host:
-        - {{ min_python }}
       run:
-        - {{ pin_subpackage("ariadne-codegen", exact=True) }}
-        - {{ min_python }}
+        - {{ pin_subpackage("ariadne-codegen", max_pin="x.x.x") }}
```

Comment:
see earlier point on #2  about `max_pin` not working when splitting `generic` and `python`

Reply:
But didn't it work? The CI ran through